pico_limbo/src/handlers/configuration.rs

```diff
@@ -229,6 +232,34 @@ fn send_tab_list_packets(batch: &mut Batch<PacketRegistry>, server_state: &Serve
     }
 }
 
+fn send_boss_bar_packets(batch: &mut Batch<PacketRegistry>, server_state: &ServerState) {
+    match server_state.boss_bar() {
+        BossBar::Enabled {
+            title,
+            health,
+            color,
+            division,
+        } => {
+            let uuid = Uuid::new_v4();
+            let (most_sig, least_sig) = uuid.as_u64_pair();
+            let packet = BossBarPacket {
+                v1_16_uuid: uuid,
+                uuid_most_sig: most_sig,
+                uuid_least_sig: least_sig,
```

Comment:
Create a constructor that takes the uuid and the boss bar action
The constructor will be the one doing the logic of splitting the uuid into two longs